### PR TITLE
Adjust common functions

### LINKED ISSUE #147: Check & move same functions to runtime/common

### PR DESCRIPTION
**Pull request by jiguantong**
Close #147

### CODE REVIEW

File: core/primitives/src/lib.rs [hunk 1]
```diff
@@ -80,3 +80,11 @@ pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
 pub const HOURS: BlockNumber = MINUTES * 60;
 /// 14,400 blocks.
 pub const DAYS: BlockNumber = HOURS * 24;
+
+/// Deposit calculator for Darwinia.
+/// 100 UNIT for the base fee, 102.4 UNIT/MB.
+pub const fn darwinia_deposit(items: u32, bytes: u32) -> Balance {
```
Comment (jiguantong) [inline core/primitives/src/lib.rs:86]: Do we need different deposit calculators for Crab & Pangolin? @AurevoirXavier

Comment (AurevoirXavier) [reply core/primitives/src/lib.rs:86]: No. Use the same configurations for (Crab,Pangolin) and (Darwinia,Pangoro). 

This shouldn't be core primitive.

Please move to common runtime.

File: runtime/common/src/lib.rs [hunk 2]
```diff
@@ -36,3 +47,31 @@ macro_rules! fast_runtime_or_not {
...
+///   - Setting it to `1` will cause the literal `#[weight = x]` values to be charged.
+pub struct WeightToFee;
+impl WeightToFeePolynomial for WeightToFee {
+	type Balance = Balance;
+
+	fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
+		// in Rococo, extrinsic base weight (smallest non-zero weight) is mapped to 1 MILLIUNIT:
+		// here, we map to 1/10 of that, or 1/10 MILLIUNIT
+		let p = MILLIUNIT / 10;
+		let q = 100 * Balance::from(ExtrinsicBaseWeight::get().ref_time());
```
Comment (AurevoirXavier) [inline runtime/common/src/lib.rs:69]: The formula might be different for different runtimes.

But I'm not sure now.

File: runtime/common/Cargo.toml [hunk 3]
```diff
@@ -11,7 +11,8 @@ version     = "6.0.0"
 
 [dependencies]
 # crates.io
-codec = { package = "parity-scale-codec", version = "3.2", default-features = false }
+codec    = { package = "parity-scale-codec", version = "3.2", default-features = false }
+smallvec = { version = "1.10" }
```
Comment (AurevoirXavier) [inline runtime/common/Cargo.toml:15]: Can we remove the smallvec which is in old files?

**Review by AurevoirXavier — APPROVED** [review]

**Pull request merged**